LiskHQ/lisk-sdk
# Update genesis block and generator keys to be read from file - Closes #7460

### What was the problem?

This PR resolves #7460 and resolves #7108

### How was it solved?
- Update config structure
- Check ABI does not send `assets` for genesis
- Update `start` and `config:show` command to use the updated config
- Update application and engine to read genesis block from file or config
- Updated generator to read keys from file if set


### How was it tested?

- check dpos mainchain example for the configs
- check dpos mainchain example for for sending transaction
-
